### ISSUE TITLE
Slow start (even with Platinum Searcher)

### ISSUE THREAD

**Issue by skippednote** [junegunn/fzf#914]
- Category
    - [x] fzf binary
    - [ ] fzf-tmux script
    - [ ] Key bindings
    - [ ] Completion
    - [x] Vim
    - [ ] Neovim
    - [ ] Etc.
- OS
    - [ ] Linux
    - [x] Mac OS X
    - [ ] Windows
    - [ ] Windows Subsystem for Linux
    - [ ] Etc.
- Shell
    - [x] bash
    - [ ] zsh
    - [x] fish

I'm using Fish shell on macOS and every time I run `fzf` in my terminal even with 5-6 empty files it takes 3-4 seconds to load. I remember I didn't have to see the loading spinner (/ 0/0) but now something has changed and I can't seem to debug it.

**Comment by junegunn** [comment]
Do you see the same delay when you do `seq 100 | fzf`? If you don't, it is probably an issue of your shell. fzf internally uses `$SHELL -c "..."' to start the command. Also test `env FZF_DEFAULT_COMMAND='seq 100' fzf`.

**Comment by skippednote** [comment]
`seq 100 | fzf` is instantaneous. 
`FZF_DEFAULT_COMMAND='seq 100' fzf` is slow, take 2-3 seconds and I tried this in bash and fish shell.

`it is probably an issue of your shell`
my `bash` has no custom settings. What could be the issue then?

**Comment by junegunn** [comment]
Then you'll see the same issue with `$SHELL -c 'seq 100'`. For some reason, your shell is not starting up quickly. You might want to check which configuration is causing the delay.

**Comment by skippednote** [comment]
Here is a short video for demonstration https://youtu.be/A3UFBgYJ0ZQ

**Comment by junegunn** [comment]
Yeah, it's really an issue of your shell configuration, and there's not much I can help with. You might want to fix the performance of `$SHELL -c 'seq 100'`.

**Comment by skippednote** [comment]
I was debugging my shell configuration as you suggested and found the issue.
I'm using Fish Shell with oh-my-fish and a few oh-my-fish plugins. I uninstalled all the plugins and installed them one by one monitoring the performance. 

`$SHELL -c 'seq 100'` was instantaneous without oh-my-fish and with it installed. Worked well with all the plugins installed as well. But at last I found that [plugin-nvm](https://github.com/derekstavis/plugin-nvm) was the issue.
